doc/source/serve/batching-guide.md

```diff
@@ -0,0 +1,46 @@
+# Requests Batching
+
+Serve offers a request batching feature that can improve your service throughput with sacrificing latency.
```

Comment:
Serve offers a request batching feature that can improve your overall service throughput by sacrificing some latency.

Reply:
I think the intention is "without sacrificing latency" ?

Reply:
@simon-mo please add a few sentences describing why batching is important for ML models (efficient vectorized computation)

Reply:
It might be subtle, the worst-case latency of a request will increase because it'll be waiting for a batch instead of being processed instantly.  But if the overall throughput increases, the average latency would decrease because there won't be as long of a queue in front of each replica.  Maybe we just omit the latency part from this first sentence